documentation/chatmeleon-swagger.yaml

```diff
@@ -267,26 +320,77 @@ components:
         createdAt:
           type: string
           format: date-time
-          description: The timestamp when the conversation was created.
-        lastMessageAt:
+          description: Date and time when the conversation was created
+        name:
+          type: string
+          description: Optional name for the conversation
+        lastActive:
           type: string
           format: date-time
-          description: The timestamp of the last message in the conversation.
-        name:
+          description: Date and time of the last activity in the conversation
+        lastMessageId:
           type: string
-          description: The name of the conversation.
+          format: uuid
+          description: References the latest message in the conversation
         isGroup:
           type: boolean
-          description: Indicates whether the conversation is a group conversation or not.
-        messagesIds:
+          description: Indicates whether the conversation is a group chat
+        groupAvatar:
+          type: string
+          description: Optional avatar for group conversations
+        nicknames:
           type: array
           items:
-            type: string
-            pattern: "^[0-9a-f]{24}$"
-          description: An array of MongoDB _ids of messages associated with this conversation.
+            $ref: "#/components/schemas/Nickname"
+          description: Nicknames for users within the conversation
         userIds:
           type: array
           items:
             type: string
-            pattern: "^[0-9a-f]{24}$"
-          description: An array of MongoDB _ids of users associated with this conversation. These user IDs can be used to reference the User model.
+            format: uuid
+            description: List of user IDs belong to the conversation
+        messageIds:
+          type: array
+          items:
+            type: string
+            format: uuid
+            description: List of message IDs belong to the conversation
+    # User_Relationship model
+    User_Relationship:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: Unique identifier for each relationship
+        type:
+          type: string
+          enum:
+            - FRIEND
+            - BLOCK
+            - FOLLOWING
+            - PENDING_FRIEND_REQUEST
+            - SENDING_FRIEND_REQUEST
```

Comment:
About following, is there any enum value for the followed person? I think there should be a value for it.

Reply:
It will in the future but for the current state of our app, this is sufficient